esp/services/ws_access/ws_accessService.cpp

```diff
@@ -2087,24 +2089,36 @@ bool Cws_accessEx::onResourceDelete(IEspContext &context, IEspResourceDeleteRequ
                 if(prefix && *prefix)
                     namebuf.insert(0, prefix);
 
-                secmgr->deleteResource(rtype, namebuf.str(), basednReq->getBasedn(), context.querySecureContext());
+                try
+                {
+                    secmgr->deleteResource(rtype, namebuf.str(), basednReq->getBasedn(), context.querySecureContext());
+                }
+                catch(IException* e)
+                {
+                    StringBuffer emsg;
+                    e->errorMessage(emsg);
+                    errMsgs.appendf("\n%d %s", e->errorCode(), emsg.str());//accumulate error messages/return codes
+                    iLastRC = e->errorCode();
```

Comment:
I would also consider when there are more than one exceptions having a new return code that means "multiple errors".  Then the caller will know they can iterate through them all.